docs/source/llms/llama-index/index.rst

```diff
@@ -38,17 +38,26 @@ The integration of the LlamaIndex library with MLflow provides a seamless experi
 Getting Started
 ---------------
 
-In this introductory tutorial, you will learn the most fundamental components of LlamaIndex and how to leverage the integration with MLflow to store, retrieve, and 
-use an index. 
+In these introductory tutorials, you will learn the most fundamental components of LlamaIndex and how to leverage the integration with MLflow to bring better maintainability and observability to your LlamaIndex applications.
 
 .. raw:: html
 
     <section>
         <article class="simple-grid">
+            <div class="simple-card">
+                <a href="notebooks/llama_index_workflow_tutorial.html">
+                    <div class="header">
+                        LlamaIndex Workflows with MLflow
+                    </div>
+                    <p>
+                        Get started with MLflow and LLamaIndex by building a simple agentic Workflow. Learn how to log and load the Workflow for inference, as well as enable tracing for observability.
+                    </p>
+                </a>
+            </div>
             <div class="simple-card">
                 <a href="notebooks/llama_index_quickstart.html">
                     <div class="header">
-                        LlamaIndex Quickstart
+                        Building Index with MLflow
```

Comment:
I think quickstart should be somewhere in the name. I also think this card should be first, as it's the "entry point" to the flavor

Reply:
We're thinking of de-emphasizing the limited scope of the index-focused low-level APIs in favor of the workflow APIs (it seems like this is the direction that the library's going to be taking) so we decided to swap these two with the workflows API taking center stage.